src/src/main/java/dao/impl/UsersDAO.java

```diff
@@ -0,0 +1,34 @@
+package dao.impl;
+
+import dao.UsersDAOInterface;
+import service.DataBaseService;
+import model.User;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
+public class UsersDAO implements UsersDAOInterface {
+
+    public UsersDAO() {
+    }
+
+    public static User create(String email, String login, String password) {
```

Comment:
why this method is static? 
maybe better without create method, we can use only method add?